e2e/pages/events/enterOthersEvent.page.js

```diff
@@ -47,7 +47,11 @@ module.exports = {
     I.selectOption(this.fields(elementIndex).gender, other.gender);
     I.fillField(this.fields(elementIndex).birthPlace, other.birthPlace);
     within(this.fields(elementIndex).address, () => {
-      postcodeLookup.lookupPostcode(other.address);
+      if (elementIndex === 'firstOther') {
```

Comment:
A bit inconsistent - in the previous file (enterChildrenEvent.page.js) the elementIndex was index and compared to 0. Here we have got some string, shouldn't this be named differently? (I know you haven't introduced this variable, but as this is refactoring change, maybe would be worth improving?)

Reply:
A `firstOther` forms part of the locator which is driven by data structures in CCD. This is old structure with `{ firstX: {}, additionalX: [{}, {}] }`. In two other places we have just a collection that when CCD UI is rendered uses item position (0-x) to uniquely name DOM elements.

Things like `first / additional` should disappear with time.